### ISSUE TITLE
Load CA's from mtls from directory not only file

### ISSUE THREAD

**Issue by hmilkovi** [haproxy/haproxy#1476]
### Your Feature Request

Hi team,
I have use case where there are many different Certificate Authorities and I think it whould be great to
handle them in directory each in it's file like ssl certificates and also expose it via Data Plane API.

### What are you trying to do?

Make mtls Haproxy configuration like:
```
bind x.x.x.x:443 ssl crt ./some.pem verify optional ca-dir /etc/ssl/service-ca-s/
```

Also like we have:
https://www.haproxy.com/documentation/dataplaneapi/community/#post-/services/haproxy/storage/ssl_certificates
it whould be nice to have the same for mtl CA certs.

### Output of `haproxy -vv`

```plain
Sorry will not disclose.
```

**Comment by wlallemand** [comment]
Will be released with 2.6, the above patch is the implementation of the feature with "ca-file". More changes are coming to select easily the system CAs on a server line.

**Comment by hmilkovi** [comment]
Great can't wait :)

**Comment by wlallemand** [comment]
Don't hesitate to test and comment before the release!

**Comment by wlallemand** [comment]
merged since 2.6, closing.
